view/search/facet-link.phtml

```diff
@@ -7,8 +8,19 @@
  * @var string $url
  * @var int $count
  */
+$this->headScript()->appendFile($this->assetUrl('js/facets.js', 'Search'));
 ?>
-<span class="<?php echo $active ? 'active' : 'inactive'; ?>">
-    <a href="<?php echo $this->escapeHtml($url); ?>"><?= $this->searchFacetValue($name, (string) $value) ?></a>
-    (<?php echo $count; ?>)
-</span>
+<div>
+    <span class="<?php echo $active ? 'active' : 'inactive'; ?>">
+        <input type="checkbox"
+            id="checkbox_<?php echo $value; ?>"
+            name="selectedFacets[<?php echo $this->escapeHtml($name); ?>][]"
+            value="<?php echo $this->escapeHtml($value); ?>"
+            data-facet-name="<?php echo $this->escapeHtml($name); ?>"
+            data-facet-value="<?php echo $this->escapeHtml($value); ?>" />
+        <label for="checkbox_<?php echo $value; ?>">
+            <a href="<?php echo $this->escapeHtml($url); ?>"><?= $this->searchFacetValue($name, (string) $value) ?></a>
+            (<?php echo $count; ?>)
+        </label>
```

Comment:
`$value` can contain space and it is invalid to have space in the id attribute.

I think can remove the `id` and `for` attributes, and instead put the checkbox inside the label element.